gpu/vars/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 # vars file for gpu
-home_dir: "/root/ansible.tmp/"
+install_dir: "/root/{{ random_name }}/"
```

Comment:
@vigneshv-ocl Let's keep `/root/ansible.tmp` instead of `/root/{{ random_name }}/` to ensure a consistent temporary directory.

Random names are only beneficial if Ansible properly cleans them up; otherwise, they can lead to leftover files accumulating over time.
```
root@node1:~# ls -latr | grep ansibl
drwx------  3 root root 4096 Feb 26 09:26 .ansible
drwxr-xr-x  2 root root 4096 Feb 27 09:36 ansible-5s"K
drwxr-xr-x  2 root root 4096 Feb 27 10:27 ansible-Gq~x
drwxr-xr-x  2 root root 4096 Feb 27 10:43 ansible-a?0\
root@node1:~# 
```